docker-compose.yaml

```diff
@@ -0,0 +1,22 @@
+version: "3.9"
+
+services:
+  postgres:
+    image: "postgres:16.1-bookworm"
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: "dummyuser"
+      POSTGRES_PASSWORD: "dummypassword"
+      POSTGRES_DB: "dummydb"
+    volumes:
+      - "pg-data:/var/lib/postgresql/data"
+  golang:
```

Comment:
To allow the code to be restarted without rebuilding `Dockerfile`, we can use this trick:
1) Set image to `golang:1.21.6-bookworm`
2) Unset `build`.
3) Set `working_dir` as `/app`
4) In `volumes`, set `.:/app`
5) In command, set `go run main.go`

Bonus points:
- Create new `make` command to build `main.go`: `go build -o build/grpc main.go`